### ISSUE TITLE
After creating a new mpe room the created room doesnot appears in the library

### ISSUE THREAD

**Issue by prastoin** [AdonisEnProvence/MusicRoom#369]
Atm after creating a mpe room the user is redirected to the library mpe search engine. ( Would we like the user to be redirected to the created room view ? )
It looks like the library mpe search engine is not refreshing it's state on rerender ?

**Comment by Devessier** [comment]
We are redirecting the user to the library screen right after we sent the information of the MPE room to create (https://github.com/AdonisEnProvence/MusicRoom/blob/f7419bf72680b8c23851480facb83028f92c6ac3/packages/client/machines/appMusicPlaylistsMachine.ts#L745).
I think the redirection should be done only after we know the server acknowledged the creation. And when we perform the redirection, we provide a parameter such as `refresh` that will tell the screen to refresh its state.

**Comment by prastoin** [comment]
In a first time I think we should not juste redirect the user to the library search engine screen but rather on the created mpe room view
Also we should redirect after the room creation server ack to do not display a not already existing playlist actor mpe room view

**Comment by Devessier** [comment]
Right!

**Comment by prastoin** [comment]
https://github.com/AdonisEnProvence/MusicRoom/pull/401
after mpe room creation the user is now redirected toward the mpe library search engine just after emitting the create client socket event, on server ack the user is redirected to the created mpe room view

Devessier has implemented previously the pull to refresh on the mpe library search engine. Could we determine that this issue could be closed ? du to this new feature that allows the user to refresh the search results ?

I'm not sure that we want to refresh the search engine result on every rerender, do we ?

**Comment by prastoin** [comment]
Indeed I think this is an issue :thinking: we should find a way to force refresh the search results do we ?

**Comment by Devessier** [comment]
> Indeed I think this is an issue 🤔 we should find a way to force refresh the search results do we ?

I think too!

**Comment by prastoin** [comment]
Closing from now have we consider this is not a priority
This might become one in the future if we face a real UX UI issue later
